src/main/java/com/paneedah/mwc/equipment/EquipmentFactory.java

```diff
@@ -0,0 +1,10 @@
+package com.paneedah.mwc.equipment;
+
+import com.paneedah.weaponlib.ModContext;
+
+public interface EquipmentFactory {
+
+    void createEquipment(ModContext modContext);
+
+}
```

Comment:
Remove modContext from this, to access a `ModContext` uses `MWC.modContext`

Reply:
Done